sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatClientBuilder.java

```diff
@@ -251,7 +251,7 @@ private HttpPipeline createHttpPipeline(HttpClient httpClient,
     }
 
     private void applyRequiredPolicies(List<HttpPipelinePolicy> policies, HttpPipelinePolicy authorizationPolicy) {
-        policies.add(new UserAgentPolicy(null, "azure-communication-chat", "1.0.0"));
+        policies.add(new UserAgentPolicy(null, "azure-communication-chat", "1.1.0-beta.2"));
```

Comment:
We should avoid using hard-coded strings for things like these. I'd recommend introducing a constant or something of the sort.